From fe005e2b4d3495ea706f76ed4c3c4e617ee7cd5f Mon Sep 17 00:00:00 2001
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Tue, 28 Jul 2009 18:42:50 +0530
Subject: [PATCH 0504/1092] cppi41: change number of DMA BLOCK from 4 to 1

Number of DMA block is different for different platforms.

Changing this to '1' for AM3517 platform.

Signed-off-by: Ajay Kumar Gupta <ajay.gupta@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 drivers/usb/musb/cppi41.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/usb/musb/cppi41.h b/drivers/usb/musb/cppi41.h
index 2b7b65c..9dbc38a 100644
--- a/drivers/usb/musb/cppi41.h
+++ b/drivers/usb/musb/cppi41.h
@@ -281,9 +281,9 @@ struct cppi41_teardown_desc {
 
 #define CPPI41_MAX_MEM_RGN		16
 
-/* CPPI 4.1 configuration for DA8xx */
+/* CPPI 4.1 configuration for AM3517 */
 #define CPPI41_NUM_QUEUE_MGR		1	/* 4  max */
-#define CPPI41_NUM_DMA_BLOCK		4	/* 64 max */
+#define CPPI41_NUM_DMA_BLOCK		1	/* 64 max */
 #define cppi41_num_queue_mgr	CPPI41_NUM_QUEUE_MGR
 #define cppi41_num_dma_block	CPPI41_NUM_DMA_BLOCK
 
-- 
1.7.11.2

